deployment/scripts/install.sh

```diff
@@ -286,22 +286,22 @@ fi
 echo "(Re-)Creating deployed objects"
 if [ "${KEEP_SUPPORT}" != true ]; then
 	oc process logging-support-template | oc delete -f - || :
-	oc delete serviceaccount,service,route --selector logging-infra=support
+	oc delete imagestream,service,route --selector logging-infra=support
```

Comment:
sure. ah, what's the "gotcha"? :) that ISs are normally recreated? that you
can remove the label to prevent this?

On Mon, Apr 18, 2016 at 12:08 PM, Eric Wolinetz <notifications@github.com>
wrote:

> In deployment/scripts/install.sh
> <https://github.com/openshift/origin-aggregated-logging/pull/117#discussion_r60086433>
> :
>
> > @@ -286,22 +286,22 @@ fi
> >  echo "(Re-)Creating deployed objects"
> >  if [ "${KEEP_SUPPORT}" != true ]; then
> >  	oc process logging-support-template | oc delete -f - || :
> > -	oc delete serviceaccount,service,route --selector logging-infra=support
> > +	oc delete imagestream,service,route --selector logging-infra=support
>
> okay that makes sense; maybe we can add a quick documentation 'gotcha' ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin-aggregated-logging/pull/117/files/e4b187c1c050cdab426a760811c479128dbc0380..586c2f42e48f04a39ece6c834162112673fd9cdc#r60086433>
>